pinot-integration-tests/src/test/java/org/apache/pinot/integration/tests/BaseClusterIntegrationTestSet.java

```diff
@@ -146,6 +146,19 @@ private void testHardcodedQueriesCommon()
     String query;
     String h2Query;
 
+    // SUM result will overflow INTEGER
+    query = "SELECT SUM(ActualElapsedTime) FROM mytable";
+
+    // SUM result will overflow INTEGER
+    query = "SELECT SUM(CAST(ActualElapsedTime AS FLOAT)) FROM mytable";
+
+    // SUM result will overflow INTEGER
+    query = "SELECT SUM(CAST(ActualElapsedTime AS BIGINT)) FROM mytable";
+
+    // SUM result will overflow INTEGER
+    query = "SELECT SUM(CAST(ActualElapsedTime AS DOUBLE)) FROM mytable";
+
+    testQuery(query);
```

Comment:
Is there a runtime test that verifies SUM (and other aggregations) on BIG_DECIMAL column? I remember testing this long back and there were some issues. Not sure if they've been fixed now.

Reply:
not really. given the fact that we are currently working around the type-hoisting behavior to not do any automatic conversion. it should be same type in same type out